README.md

```diff
@@ -8,31 +10,25 @@ Exposing DanielSWolf / rhubarb-lip-sync as a service
 docker build -t korerorero-mouth-shapes:latest .
 ```
 
-To run
+__To run__
 
 ```bash
 docker run -p 3000:3000 korerorero-mouth-shapes:latest
```

Comment:
Hmmm, this command doesn't work on my machine. I get this error:
![image](https://user-images.githubusercontent.com/13788498/80174530-fa190d80-8646-11ea-991f-f94d83e548db.png)
Is it because of the .env file again?